### PR TITLE
Remove Bert tokenizer dependency from DistillBert (slow/fast) tokenizers

### PR DESCRIPTION
**Pull request by IvanLauLinTiong**
Hi @sgugger,

Fixes https://github.com/huggingface/transformers/issues/19303
- The `BertTokenizer` dependency has been removed from `DistillBerTokenizer`
- The `BertTokenizerFast` dependency has been removed from `DistillBerTokenizerFast`

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@sgugger

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Very clean, thanks a lot for working on this!

File: src/transformers/models/distilbert/tokenization_distilbert.py [hunk 1]
```diff
@@ -59,18 +64,477 @@
...
+            mask_token=mask_token,
+            tokenize_chinese_chars=tokenize_chinese_chars,
+            strip_accents=strip_accents,
+            **kwargs,
+        )
+
+        if not os.path.isfile(vocab_file):
+            raise ValueError(
+                f"Can't find a vocabulary file at path '{vocab_file}'. To load the vocabulary from a Google pretrained"
+                " model use `tokenizer = BertTokenizer.from_pretrained(PRETRAINED_MODEL_NAME)`"
```
Comment (SamuelzXu) [inline src/transformers/models/distilbert/tokenization_distilbert.py:169]: @sgugger Should this error message raise "DistilBertTokenizer.from_pretrained"....etc?

Comment (IvanLauLinTiong) [reply src/transformers/models/distilbert/tokenization_distilbert.py:169]: @SamuelzXu 
Good catch. Maybe you could help me to create a new PR to fix that?
Thanks

Comment (sgugger) [reply src/transformers/models/distilbert/tokenization_distilbert.py:169]: Indeed!

Comment (SamuelzXu) [reply src/transformers/models/distilbert/tokenization_distilbert.py:169]: Sure thing

**Pull request merged**